### ISSUE TITLE
Change EFS Plugin to prioritize security group from plugin config over instance meta-data

### ISSUE THREAD

**Issue by febbraro** [codedellemc/libstorage#511]
# Summary
When installing the EFS plugin you can configure a security group using the EFS_SECURITYGROUPS variable. Use this configuration as higher priority than the security group(s) provided by the instance.

# New Feature
If the instance has only one security group, it will use that instance group by default. If it used config as higher priority, it could alert the user is they have not assigned the security group to the instance.

## Steps To Reproduce
A way to see this...

* Create SecurityGroup1 (with no NFS Port config)
* Create SecurityGroup2 (with NFS port config)
* Create an instance and assign it SecurityGroup1 (and don't assign it SecurityGroup2)
* Install the rexray/efs plugin and configure EFS_SECURITYGROUPS with SecurityGroup2
    * `docker plugin install rexray/efs EFS_SECURITYGROUPS="sg-XXXXXX"`
* Launch a docker stack with a rexray/efs volume mounted to an instance
* When the EFS Mount point is created it will have SecurityGroup1 assigned to it.

Here is a compose file that can be used via `docker stack deploy`

```yaml
version: '3.2'

services:
  test:
    image: alpine
    volumes:
      - files:/var/test/files

volumes:
  files:
    driver: rexray/efs
    driver_opts:
      size: 10
```

**Comment by febbraro** [comment]
@akutz This is what you helped me debug. THANKS!!

**Comment by akutz** [comment]
Hi @febbraro,

Do you remember what we did for this?

**Comment by febbraro** [comment]
@akutz Posting a summary of our Slack convo here (that summarizes and IRL debugging session)

I had an instance with a default SG and an SG for EFS (not attached to said instance) but defined in my IAM account.

I specified the EFS SG (the unassigned one) to the plugin install command  Since the instance only had 1 SG (not the EFS enabled one, but the default) it used that default SF regardless (by current design to use the only SG on an instance if only one exists)

We got around my problem, by attaching that EFS SG to the instance, then the instance had 2 SGs and the one I wanted the EFS plugin to use was there, so it would use it.

I think the fix we discussed was if the SG configured in the plugin was not present on the instance, tell the user as much to make debugging easier.

Hope this helps.
